client/src/pages/Profiles/ProfileListings/ProfileListing.tsx

```diff
@@ -100,12 +106,12 @@ export default function ProfileListing({}: Props): ReactElement {
                         </Grid>
                         <Grid item xs={4}>
                           <Typography className={classes.profileAddress} color="text.secondary">
-                            {`${profile.address}`}
+                            {`${profile.location}`}
                           </Typography>
                         </Grid>
                         <Grid item xs={6}>
                           <Typography sx={{ fontWeight: 1000 }} className={classes.profilePay} color="text.secondary">
-                            {`${profile.pay}`}
+                            {profile.payRate ? `${profile.payRate}$ /hr` : ''}
```

Comment:
When we query the db for sitter profiles, we should also check whether than sitter has a `payRate` - we return their profile only in case they have a rate

Reply:
I will address this in my PR that has full integration of the profile listings including the landing page and the dashboard searchbar, good point